DNN Platform/DotNetNuke.ContentSecurityPolicy/BaseCspContributor.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.ContentSecurityPolicy
```

Comment:
Is this the same as the pending CSP PR into `develop`?

Reply:
Yes it's the same. It need to be removed from here after the pr in develop is merged, so we can remerg it in this branch.
Or some other sujestion how to manage this ?